City-of-Helsinki/hel-fi-drupal-grants
# UHF-9944 packages

# [UHF-9944](https://helsinkisolutionoffice.atlassian.net/browse/UHF-9944)
Checked composer packages

## What was done
Check commits

## How to install

* Make sure your instance is up and running on correct branch.
  * `git checkout UHF-9944_packages`
  * `make fresh`
* Run `make drush-cr`

## How to test
- Install site normally
- Check that everything works, especially the features using updated packages


[UHF-9944]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-9944?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
